unit-tests/src/main/scala/tests/NativeFramework.scala

```diff
@@ -10,6 +10,30 @@ class NativeFramework extends Framework {
 
   override def runner(args: Array[String],
                       remoteArgs: Array[String],
-                      testClassLoader: ClassLoader): Runner =
+                      testClassLoader: ClassLoader): Runner = {
     new NativeRunner(args, remoteArgs)
+  }
+
+  /** Scala Native specific: Creates a worker for a given run.
```

Comment:
Here we can also remove the big comment, and inherit it from `sbt.testing.Framework` instead.